Dockerfile

```diff
@@ -23,17 +23,26 @@ COPY main.go main.go
 COPY apis/ apis/
 COPY controllers/ controllers/
 COPY resources/ resources/
+COPY must-gather/ must-gather/
 
 # Build
 RUN VERSION=${BUILD_VERSION} GIT_COMMIT=${GIT_COMMIT} GIT_VERSION=${GIT_VERSION} make build
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# Use origin image for oc cli binary
+FROM quay.io/openshift/origin-cli:4.12 AS origincli
+
+# Use Red Hat Universal Base Image Micro as minimal base image to package the manager binary.
+# See https://catalog.redhat.com/software/containers/ubi8/ubi-micro/5ff3f50a831939b08d1b832a
+# TODO: Use authenticated registry: registry.redhat.io
+FROM registry.access.redhat.com/ubi8/ubi-micro:8.7-6
```

Comment:
Should the authenticated registry be used instead?

Reply:
We use currently use registry.redhat.io/ubi8/ubi-minimal:latest
see:
https://gitlab.cee.redhat.com/custom-metrics-autoscaler/cpaas-midstream/-/blob/ocma-2-rhel-8/distgit/containers/custom-metrics-autoscaler-operator/Dockerfile.in

Reply:
Addressed in https://github.com/openshift/custom-metrics-autoscaler-operator/pull/21/commits/f266c048f1a03f55f5459fec2b5e02f7e4edf428